pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -76,6 +79,21 @@ public void writeResource(HapiRowDescriptor element)
             .setVersionId(element.resourceVersion())
             .setLastUpdated(simpleDateFormat.parse(element.lastUpdated()));
 
+    if (element.getTags() != null) {
+      List<Coding> tags = new ArrayList<>();
+      for (ResourceTag resourceTag : element.getTags()) {
+        // The column tag_type of value 0 means it's of type TAG, 1 for PROFILE and 2 for SYSTEM.
```

Comment:
Might be worth adding a pointer to the HAPI documentation for this, if there is any.

Reply:
Yes, I could not find anything relevant to SYSTEM tags

Reply:
Added documentation link

Reply:
system related piece has been addressed, thanks